phpcs.xml.dist

```diff
@@ -59,6 +59,8 @@
 	<exclude-pattern>/src/wp-includes/class-requests\.php</exclude-pattern>
 	<exclude-pattern>/src/wp-includes/class-simplepie\.php</exclude-pattern>
 	<exclude-pattern>/src/wp-includes/class-snoopy\.php</exclude-pattern>
+	<exclude-pattern>/src/wp-includes/class-wp-block-parser\.php</exclude-pattern>
```

Comment:
```suggestion
```

Why is this file being excluded?

Reply:
hmm, good question! I'm not sure and it looks extraneous, I will try removing that line.